### ISSUE TITLE
Multiplayer Server Issue

### ISSUE THREAD

**Issue by Daymond2253** [jakobkmar/level-border#5]
Hi, My friend and I create a server and use level-border. And put it into shared, so we all have the same level. But prob lame comes when one of them die. For the example when one of us die server get stuck and all player freeze. What I think is when one player die server or your mod get confused! He can't calculate level because we have shared level. Because when we have 4 level and the died player have 0 level and your mod is in shared mode. He gets confused what level should I give him, 4 or 0. And get to Crash or Freeze.

**Comment by CrazySaft** [comment]
Yes i experienced that too, if one player leaves and he is lvl 20 and i stay online and grind till lvl 60 the plugin gives me  again 20 if he joins. So for this I just turned the mode ito SUM which just adds all player levels together and then i doesn't matter what lvl you are or die.

**Comment by Daymond2253** [comment]
> Yes i experienced that too, if one player leaves and he is lvl 20 and i stay online and grind till lvl 60 the plugin gives me again 20 if he joins. So for this I just turned the mode ito SUM which just adds all player levels together and then i doesn't matter what lvl you are or die.

Well I tried that too but for me, it's too easy & no fun for me.

**Comment by CrazySaft** [comment]
Yes i agree but i had no choice lol

**Comment by jakobkmar** [comment]
Cannot reproduce, tested death and disconnect cases, both worked for me. Do you have a more specific example where this happens?

**Comment by ShubhSarin** [comment]
we both are playing together and server is crashing, here's the video [https://drive.google.com/file/d/1crFuDbaBINYTjeXrq5u12hdfS38EDjpp/view?usp=sharing](url)

**Comment by jakobkmar** [comment]
Thank you for the demo video.

If the server crashed, there must be an error message in the console, are you able to share that?
